Update satellite and pcap for AWS (#151)
levoai/docs: docs/api-observability/install-guide/install-sensor/pcap-sensor.md

 - The profile should have all the required permissions as listed [here](#aws-permissions)
 - Refer [pcap-filter-guide](https://www.tcpdump.org/manpages/pcap-filter.7.html) to apply filters.

> **_NOTE:_**  You need to have the satellite installed already to configure the sensor to point to it. If you haven't done it already, head over to [Install Satellite ](../install-satellite.mdx)

 ## Follow instructions for your platform

 - [Install on Fargate](#install-fargate)
 - [Install via Docker](#install-docker)
 - [Install on Kuberenetes](#install-kubernetes)

<a id="install-fargate"></a>

## Install Sensor on Fargate

> The pcap Sensor can be installed as a sidecar on a already present AWS task, and can be added to its task definition via the AWS Console.

Reviewer: ```suggestion
The pcap Sensor can be installed as a sidecar on an existing AWS task by adding to it's task definition via the AWS Console.
```
